week-14/practices/12-context-practices/practice-1-README.md

```diff
@@ -87,24 +94,58 @@ can destructure:
   setCurrentSign: *function*
 }
 ```
+Now that we have our `currentSign`, we will need to grab some details about this
+sign. We have already created data for you to use, which is located in
+`src/data/horoscopes.js`. Take a second to familiarize how the data is setup,
```

Comment:
**Line 99:** Would "set up" make more sense than "setup"? 

I'm asking, not telling. I'm honestly not sure

Reply:
I was battling with this as well LOL I thought set up makes sense, but went with setup. 👀

Reply:
I'm thinking "set up" too

Reply:
consider it changed!